src/components/CardItem/CardItem.tsx

```diff
@@ -21,54 +19,99 @@ interface CardItemInterface {
 
 function CardItem({ item, type }: CardItemInterface) {
   const { t } = useTranslation();
+  const theme = useTheme();
+  const mobile = useMediaQuery(theme.breakpoints.down('sm'));
 
   return (
     <Box>
       {type === 'Possibilities' && (
-        <Box
-          maxWidth="18.063rem"
+        <Stack
           sx={{
             height: '100%',
-            py: '2.5rem',
-            pl: '1.563rem',
-            pr: '2rem',
+            padding: {
+              md: '34px 30px 25px',
+              lg: '2.5rem 1.563rem 2rem 2.5rem',
+            },
             borderRadius: '0.75rem',
             border: '1px solid #EAECEE',
+            flexDirection: { md: 'row', lg: 'column' },
+            alignItems: { md: 'center', lg: 'start' },
+            justifyContent: { md: 'space-between', lg: 'flex-start' },
           }}
         >
-          <Image
-            src={item.imageSrc}
-            alt="Card Image"
-            width={item.width}
-            height={item.height}
-          />
-          <UITypography variant="h6" sx={{ pt: '32px' }}>
-            {t(item.title)}
-          </UITypography>
-          <UITypography variant="bodyText16" sx={{ mt: '0.75rem' }}>
-            {t(item.text)}
-          </UITypography>
-        </Box>
+          <Box>
+            <Image
+              src={item.imageSrc}
+              alt="Card Image"
+              width={item.width}
+              height={item.height}
+            />
+          </Box>
+          <Stack maxWidth="294px">
+            <UITypography
+              variant="h6"
+              sx={{ pt: { md: '0', lg: '32px', xl: '32px' } }}
```

Comment:
for this reason, create scss files where you will put styles from this component